netutils/ftpc/ftpc_transfer.c

```diff
@@ -264,7 +267,10 @@ static FAR char *ftpc_abspath(FAR struct ftpc_session_s *session,
 
   else if (strncmp(relpath, "./", 2) == 0)
     {
-      asprintf(&ptr, "%s%s", curdir, relpath + 1);
+      if (asprintf(&ptr, "%s%s", curdir, relpath + 1) < 0)
+        {
+          ptr = NULL;
```

Comment:
why this is needed?

Reply:
As noted [here](https://github.com/apache/nuttx-apps/pull/1713#discussion_r1172763218).

Reply:
NULL-ed at line 224